### PR TITLE
Added pagination at the end of the plugin table

### PR DESCRIPTION
**Pull request by borracciaBlu**
Added pagination at the end of the plugin table.

### CODE REVIEW

**Comment by borracciaBlu** [comment]
Can someone retry the build on Travis?  It failed in downloading the plugins...

File: src/tmpl/plugins.pug [hunk 1]
```diff
@@ -20,3 +20,4 @@ block content
                   th.colDown
                     | Downloads
                     span.legend last 30 days
+
```
Comment (XhmikosR) [inline src/tmpl/plugins.pug]: This is redundant; the file has already an ending line break.

**Comment by XhmikosR** [comment]
I restarted Travis, let's see.

**Comment by XhmikosR** [comment]
Rebased and cleaned it up. After looking at this, I fail to see how it's an improvement compared to the current solution.

/CC @vladikoff for the final choice.

**Comment by XhmikosR** [comment]
Nevermind, I missed the obvious that we didn't have pagination at the bottom.

Isn't there any other way to achieve the same without changing the ads placement?

**Comment by XhmikosR** [comment]
OK, changed the patch so that it only adds the pagination at the bottom.

Waiting for @vladikoff.

**Comment by vladikoff** [comment]
this shifts the pagination on top of the search in the main view for me:



does that do that for you?

**Comment by XhmikosR** [comment]
Yeah. I guess that is why @borracciaBlu had moved the advertisement below `.brand .logo`. I restored his original patch if you want to check that out too.

**Comment by borracciaBlu** [comment]
sorry for the super long delay in answering ...

so @XhmikosR the reason why this PR is because I found myself wishing for a pagination bar at the bottom when i was searching for a plugin.

I usually work with a laptop; the screen is small and i have to scroll on top all the time to change page and this was frustrating me. 

Thus I thought a PR was better then a complain.

Of course this is opinionated. So, if you think that a pagination at the button of the page is useless no big deal, no worries :)

Sorry again for the looong delay...

**Pull request closed without merging**